user_manual/talk/advanced_features.rst

```diff
@@ -112,3 +112,44 @@ specific room.
 
 .. image:: images/talk-breakout-rooms-sidebar-item.png
 
+Call recording
+--------------
+Recording feature provides users an opportunity to:
+
+- Start and stop recordings during call.
+- Record video and audio stream of speaker, as well as screen share.
+- Access, share and download recorded files for future reference or distribution.
+
+Manage a recording
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The moderator of the conversation can start a recording together with a call start or anytime during it. To enable:
+
+- **Before the call**: tick a checkbox "Start recording immediately with the call" in Media settings, then click on "Start call".
+- **During the call**: click on the top-bar menu, then click "Start recording".
+
+.. image:: images/start-recording-before-call.png
+.. image:: images/start-recording-in-call.png
+
+The recording will start shortly, and you will see a red indicator next to the call time. By clicking on it, you can stop the recording any time.
+
+.. image:: images/stop-recording.png
+
+After stopping a recording, the server will take some time to prepare and save a recorded file.
+Moderator, who started the recording, should receive a notification about status of the file. From there it can be shared then to the conversation.
+
+.. image:: images/shared-recordings.png
+
+Recording consent
+^^^^^^^^^^^^^^^^^^^^^
+
+For compliance reasons with various privacy rights it is possible to ask participants for a consent to be recorded before joining the call.
+System administrator can force it for all conversations, or keep it configurable by moderators. Then moderator can see this option in Conversation settings:
+
+.. image:: images/enable-recording-consent.png
+
+When recording consent is enabled, every participant, including moderators, will see a pop-up window and has to give a consent before joining a call.
```

Comment:
```suggestion
When recording consent is enabled, every participant, including moderators, will see a pop-up window and has to give consent before joining a call.
```

Reply:
isn't that Media Settings ? Why is it referred to as a unknown pop-up window?

Reply:
```suggestion
If call recording consent is enabled, all participants, including moderators, will see a highlighted section in Media Settings when joining a call. 

The highlighted section will inform participants that the call may be recorded. To join the call, each participant must check the box to give explicit consent for recording. If a participant does not consent, they will not be allowed to join calls that involve recording.
```